wrapanapi/systems/openstack.py

```diff
@@ -758,8 +801,35 @@ def remove_tenant(self, tenant_name):
         tid = self._get_tenant(name=tenant_name)
         self.tenant_api.delete(tid)
 
-    def create_vm(self):
-        raise NotImplementedError('create_vm not implemented.')
+    def create_vm(self, name, image_name=None, image_id=None, assign_floating_ip=False,
```

Comment:
I'm wondering what it is for. we already have deploy method which helps us to deploy instance from image. I guess this one just duplicates functionality.